Tests/AdyenTests/Card Tests/Validators/CardExpiryDateValidatorTests.swift

```diff
@@ -24,11 +24,8 @@ class CardExpiryDateValidatorTests: XCTestCase {
         // Test date 30 years in the future from reference date 01/01/2020.
         XCTAssertTrue(validator.isValid("0150"))
         
-        // Test date less than 3 months in the past from reference date 01/01/2020.
+        // Test date 3 months in the past from reference date 01/01/2020.
         XCTAssertTrue(validator.isValid("1019"))
-        
-        // Test date 3 months from reference date 01/01/2020.
-        XCTAssertTrue(validator.isValid("0919"))
```

Comment:
Instead of deleting we should update the test to reflect the month change. So if we change those numbers again these tests should fail

Reply:
As discussed, I have made the change.